examples/plot_goes_hek_m25.py

```diff
@@ -0,0 +1,32 @@
+"""
+===============================
+GOES Flare and HEK Plot Example
+===============================
+
+Here is an example showing how to combine GOES and HEK data
+"""
+
+import matplotlib.pyplot as plt
+from sunpy.lightcurve import GOESLightCurve
+from sunpy.time import TimeRange, parse_time
+from sunpy.net import hek
+
+###############################################################################
+# Let's first grab GOES XRS data for a particular time of interest
+
+tr = TimeRange(['2011/06/07 04:00', '2011/06/07 12:00'])
```

Comment:
with the `/` notation on dates I never know whether this is 6th July or 7th June... I probably know in this case because I've studied such event but I would prefer a date format less ambiguous "2011-06-07".